configs/config_params.yaml

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+
+##USER NOTE: Certain fields are commented out as they are
```

Comment:
this user note is not needed anymore, right?

Reply:
done.